packages/react/src/components/Forms/Select/mono.spec.tsx

```diff
@@ -1138,5 +1138,19 @@ describe("<Select/>", () => {
       const label = screen.getByText("City");
       expect(Array.from(label.classList)).toContain("offscreen");
     });
+
+    it("renders menu empty placeholder when there are no options to display", async () => {
+      render(
+        <CunninghamProvider>
+          <Select label="City" options={[]} hideLabel={true} />
+        </CunninghamProvider>,
+      );
+      const input = screen.getByRole("combobox", {
+        name: "City",
+      });
+      const user = userEvent.setup();
+      await user.click(input);
```

Comment:
Just a matter of convention, this is the way it is done nearly everywhere else in the codebase

Reply:
I'm also use to directly write `userEvent.click()`, but since the v13 [they advice to use setup before](https://testing-library.com/docs/user-event/setup).
```
Using the methods on the instances returned by userEvent.setup() is recommended.
```